Cargo.lock

```diff
@@ -9228,7 +9183,7 @@ checksum = "4ee73e6e4924fe940354b8d4d98cad5231175d615cd855b758adc658c0aac6a0"
 dependencies = [
  "cfg-if",
  "digest 0.10.3",
- "rand 0.8.5",
+ "rand 0.4.6",
```

Comment:
This change is strange :thinking:

Reply:
True, but that's what `cargo` outputs 🙄

Reply:
I think you should be able to revert it. I see no reason to use older version and `twox-hash` specified range of `>=0.3.10, <0.9`